### DISCUSSION TITLE
Deep Mutational Scanning Dataset - raw data and predictions

### DISCUSSION THREAD

**Discussion by oliverdutton** [facebookresearch/esm#118] [Q&A]
Hello,
I love the papers and work. In Meier et. al. (ESM1v paper) and Rives et. al. (PNAS paper, ESM1b) you reproduce the datasets used by Marks' lab for deep mutational scanning. Could you please provide a file with these datasets?

Ideal would be a csv with the columns {dataset_name, mutant, mutant_sequence, experimental_value, and the predicted values you have in the paper (evmutation, deepsequence, esm1v's, etc)}.

**Comment by Turakar** [comment]
Yes, this would be very interesting, also because I have troubles finding all the wildtype sequences and parsing the SI myself. This would ensure that we all use the same data for reproducibility.
